sympy/integrals/tests/test_integrals.py

```diff
@@ -317,6 +317,18 @@ def test_issue_7450():
     assert re(ans) == S.Half and im(ans) == -S.Half
 
 
+def test_issue_8623():
+    assert integrate(cos(x)**2 / (1 + sin(x)**2), (x, 0, pi) == pi*(sqrt(2)-1)
```

Comment:
`)` missing?

Reply:
ohk! Sorry actually something is happened with my local branch. i.e, why I'am editing it online and I forgot to add `)`.